### ISSUE TITLE
Performance issue on player startup

### ISSUE THREAD

**Issue by theweavrs** [theweavrs/BreadPlayer#107]
These are the CPU and RAM usage stats I noticed on player startup:
#### CPU: 
Max 100%
Min 25%

#### RAM:
125 MB max.

Total Songs: 1206

@Bond-009 @vsarunov @stamlercas Would like you guys to confirm.

**Comment by theweavrs** [comment]
So I changed the DB Engine from LiteDB to DBreeze and these are the stats:
#### CPU:
Max 33%
Min 13%

#### RAM:
41 MB - 84 MB

This is considerable improvement but still 33% CPU is too much, I would like it near 10% or perhaps 20% max.

#### PC Specs:
OS Name:	Microsoft Windows 10 Pro
Version:	10.0.14393 Build 14393
Processor:	Intel(R) Core(TM) i7-3770S CPU @ 3.10GHz, 3101 Mhz, 4 Core(s), 8 Logical Processor(s)
Installed Physical Memory (RAM):	16.0 GB
Total Physical Memory:	16.0 GB

**Comment by theweavrs** [comment]
So after playing around with different JSON Serializers for DBreeze, I settled upon JSON.Net which I am not happy with because its still too slow. Why didn't I use protobuf-net? Or perhaps NetJSON? The main reason is non of these officially or unofficially support UWP which is really really sad. So instead of wasting my time in wallowing, I tweaked other parts of the code and these are the results so far:

CPU:

Max 29%
Min 14%

RAM:

**67 MB - 96 MB**

PC Specs:

OS Name:	Microsoft Windows 10 Pro
Version:	10.0.14393 Build 14393
Processor:	Intel(R) Core(TM) i7-3770S CPU @ 3.10GHz, 3101 Mhz, 4 Core(s), 8 Logical Processor(s)
Installed Physical Memory (RAM):	16.0 GB
Total Physical Memory:	16.0 GB
